packages/babel-helpers/src/helpers.js

```diff
@@ -5,11 +5,7 @@ import template from "@babel/template";
 const helpers = {};
 export default helpers;
 
-// Helpers never include placeholders, so we disable placeholder pattern
-// matching to allow us to use pattern-like variable names.
-const defineHelper = template.program({ placeholderPattern: false });
-
-helpers.typeof = defineHelper(`
+helpers.typeof = () => template.program.ast`
```

Comment:
was removing `placeholderPattern` for those helper templates deliberate?

Reply:
Yeah, the `.ast` variant already returns the AST unchanged, so there aren't any placeholders to bother searching for.

Reply:
Cool, good to know. Thanks for clarifying